tests/server-multiple-retention-policies/README.md

```diff
@@ -0,0 +1,222 @@
+   ```bash
+    docker stop basic-retention_member-generator
+    ```
+
+5. Wait 10 seconds and check the number of members in the database again:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should be exactly 10.
+
+6. Remove the view:
+   ```bash
+   curl -X DELETE 'http://localhost:8080/admin/api/v1/eventstreams/mobility-hindrances/views/latestVersion'
+   ```
+
+## Test Point In Time Retention
+To execute this test scenario, run the following steps:
+
+1. Add the point in time view:
+   ```bash
+   bash ./config/post-point-in-time-view.sh
+   ```
+
+2. Start sending data to the LDES server:
+    ```bash
+    docker compose up ldes-message-generator
+    ```
+
+3. Repeatedly check the number of members in the database:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should stagnate between 0 and 10 members.
+   After 30 seconds since creating the view, the number of members should steadily keep increasing and exceed the 10 members.
+
+4. Stop sending data:
+   ```bash
+    docker stop basic-retention_member-generator
+    ```
+
+5. Wait 10 seconds and check the number of members in the database again:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should now remain constant
+
+6. Remove the view:
+   ```bash
+   curl -X DELETE 'http://localhost:8080/admin/api/v1/eventstreams/mobility-hindrances/views/pointInTime'
+   ```
+
+## Test Combined Retention
+To execute this test scenario, run the following steps:
+
+1. Add the combined view:
+   ```bash
+   bash ./config/post-combined-view.sh
+   ```
+
+2. Start sending data to the LDES server:
+    ```bash
+    docker compose up ldes-message-generator
+    ```
+
+3. Repeatedly check the number of members in the database:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should stagnate between 5 and 15 members.
+
+4. Start sending members with another version to the LDES server:
+    ```bash
+    docker compose up ldes-message-generator-2
+    ```
+   
+5. Repeatedly check the number of members in the database again:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should now increase and stagnate between 10 and 30 members.
+   After 1 minute since creating the view, the number of members should steadily keep increasing and exceed the 30 members.
+
+6. Stop sending data:
+   ```bash
+    docker stop basic-retention_member-generator
+    docker stop basic-retention_member-generator-2
+    ```
+
+7. Wait 10 seconds and check the number of members in the database again:
+   ```bash
+   bash ./config/poll-members.sh
+   ```
+   The amount of members should now remain constant.
+
+8. Remove the view:
+   ```bash
+   curl -X DELETE 'http://localhost:8080/admin/api/v1/eventstreams/mobility-hindrances/views/combined'
+   ```
+
+## Test Teardown
+To stop all systems use:
+```bash
+docker compose rm -s -f -v basic-retention_member-generator
```

Comment:
These two top commands do not work, the containers are not cleaned up properly (try running 'docker ps -a' after testing, and you will notice that the two message generators still exist). I get the following warning:
`no such service: ldes-member-generator-2`

You have to use the service name from your docker compose file:
`docker compose rm -s -f -v ldes-message-generator`
and
` docker compose rm -s -f -v ldes-message-generator-2`